### DISCUSSION TITLE
【分享】有没有帮忙调试[龙酱]自动战斗部分的以太适性高的绳匠?

### DISCUSSION THREAD

**Discussion by yokuminto** [share121/inter-knot#1558] [其他]
![5f8c897b79e4056234f8aa0e2aca929a_6508191795733777255](https://github.com/user-attachments/assets/6ee78358-a9cb-444b-a714-4cd8055da0ac)

我的以太适性很差，进不了空洞，一直在研究怎么能面对那些危险的以骸，最后找到了一个搭载了 [ **自动化战斗系统** ] 的邦布
其实从结论上看确实很成功，至少在系统的精准量控下，我再也没被打的张口闭眼过

每天新出的刮刮乐有邦布帮我刮奖，过分熬夜的话邦布也会帮忙进行店铺的经营，头上又有一对小龙角，所以我干脆喊它 [ **龙酱** ] 了

但是，我发现在面对一些特别的以骸时（好像是叫塔纳托斯，拥有诡异的瞬间移动能力，和难以捉摸的斩击模式……），[龙酱]被打的泣不成声……制作系统的绳匠表示，[龙酱]需要以太适性高，精通如何对付以骸的绳匠或盗洞客才能帮助其进行改进

我试着带着[龙酱]进行一些新领域的行为训练，希望能够对它进行改进，但是我的以太适性很差，没办法教它怎么更好的面对以骸
可我又实在不想[龙酱]看到在家每天唠唠叨叨可怜兮兮的样子，所以来这里问问，有没有人能带[龙酱]进行探索训练，顺便给它打打分，完成它的心愿？

<a href="https://github.com/DoctorReid/ZenlessZoneZero-OneDragon">领养龙酱的信箱</a>

PS：此条分享信息将会于不久后关闭

**Comment by uyfh352** [comment]
不是哥们，你来真的啊？

**Comment by yokuminto** [comment]
XD 千万不要偷偷告诉制作龙酱系统的作者是我干的
